java/test/src/main/java/io/ray/test/PlacementGroupTest.java

```diff
@@ -0,0 +1,47 @@
+package io.ray.test;
+
+import io.ray.api.ActorHandle;
+import io.ray.api.Ray;
+import io.ray.api.id.ActorId;
+import io.ray.api.placementgroup.PlacementGroup;
+import io.ray.api.placementgroup.PlacementStrategy;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+@Test
+public class PlacementGroupTest extends BaseTest {
+
+  public static class Counter {
+
+    private int value;
+
+    public Counter(int initValue) {
+      this.value = initValue;
+    }
+
+    public int getValue() {
+      return value;
+    }
+  }
+
+  public void testCreateAndCallActor() {
+    List<Map<String, Double>> bundles = new ArrayList<>();
```

Comment:
Can you add a todo comment saying that currently Java doesn't support multi-node tests.  This test just creates a placement group with one bundle. It's not comprehensive to test all placement group test cases.